examples/Workflow/WorkflowUnitTest/Usings.cs

```diff
@@ -0,0 +1 @@
+global using Xunit;
```

Comment:
Personal preference, you don't need to change this, but I don't really like global usings just cause it makes it harder to see what individual file/classes are bringing in.

Reply:
I'm fine removing it. I just left it there since VS added it, but I agree that it can make the code harder to reason about.